src/App.js

```diff
@@ -18,7 +20,12 @@ function App() {
         }
         
         { !gameStatus.loadIntro &&
-          <LearningModule setGameStatus={setGameStatus}/>
+         <div>
```

Comment:
Please add a className that represents the purpose of this div.

Reply:
Done